code/src/App.js

```diff
@@ -1,9 +1,8 @@
 import React from 'react'
 
+import MoviePage from 'components/MoviePage'
+
+
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
+  return <MoviePage />
```

Comment:
You remebered that you don't need a "<> </>" when its only 1 component, nice!

Reply:
Nice that you kept the entry point clean!